woocommerce/woocommerce-android
# Fix the Coupons M2 release

This PR adds some of the missing things for the Coupons M2 release, such as:

- Updated release notes
- Added tracking for edit coupon actions
- Updated coupon banner message
- Enabled the feature flag for release builds

**To test:**

- Verify that the coupon editing works as expected
- Verify the edit actions are tracked as listed in the [spreadsheet ](https://docs.google.com/spreadsheets/d/17cKoFpHQpUMNo-stHSIyEyHbOZ30AIikTHWf9GKVwI0/edit#gid=0)
